README.md

```diff
@@ -58,37 +63,32 @@ windows.
 
 ### Subscribe to events
 
-Subscribing to events is also simple. The following example tells Chasqui to
-forward events to this subscriber's `my-app` queue. Chasqui will detect whether
-you are using Sidekiq or Resque and generate an appropriate worker class for you.
-Within the subscriber block, you use the `on(event_name)` method to register blocks
-of code to handle that event.
+Subscribing to events is also simple. In the following example, we create a
+subscriber to handle events published to the `user.sign-up` channel.
 
     # file: subscriber1.rb
     require 'chasqui'
 
-    Chasqui.subscribe queue: 'my-app' do
+    class UserSignUpSubscriber
+      include Chasqui::Subscriber
+      subscribe channel: 'user.sign-up'
```

Comment:
@Azolo good point on looking at the redis db as the "channel".  I was calling out "channels" b/c @jbgo has you configure channels in the API.

He could rename this config option `event:` and then you could use redis dbs to "channel" different events, I guess?